src/action-sheets/index.js

```diff
@@ -128,14 +130,26 @@ const reply = {
   },
 };
 
+const quoteAndReply = {
+  title: 'Quote and reply',
+  errorMessage: 'Quote-and-reply failed',
+  action: async ({ message, composeBoxRef }) => {
+    if (!composeBoxRef.current) {
+      logging.error("quoteAndReply button pressed when it shouldn't have appeared in the UI");
```

Comment:
There's a bit of a race here: this `composeBoxRef` can be mutated when the ComposeBox rerenders, so it may have been mutated between when we produced the action sheet (and decided to show the button) and invoked this action handler. So this error can occur when there's no unknown bug.

Instead, we could have `handleOutboundEvents.js` pass the immutable value `composeBoxRef.current`, rather than the mutable `composeBoxRef`. That would eliminate this race, so that seeing this error would be informing us of a bug.

(Or it could deconstruct that and pass the individual `doQuoteAndReply` callback.)